src/app/test/publish_request_test.py

```diff
@@ -39,107 +48,116 @@ def localgarden(localgarden_system):
 
 
 class TestSubscriptionEvent(object):
-    def test_topic_one(self, monkeypatch, localgarden):
+    def test_newtopic(self, monkeypatch, localgarden):
```

Comment:
Can you make a test where it matches on both the commands topic list and the new Topic/Subscriber list, then prove that it only triggers one request? I don't see anything in your code stating that it wouldn't only do one request, but I want to future-proof us just in case we tinker with that code down the line.

Reply:
Added new test that will match on topic commands and the new topic subscribers. Triggers on commands topics first.